python/nav/web/useradmin/views.py

```diff
@@ -720,6 +720,7 @@ def post(self, request, *args, **kwargs):
             token.activates = datetime.fromtimestamp(claims['nbf'], tz=timezone.utc)
             token.hash = hash_token(encoded_token)
             token.save()
+            messages.success(self.request, 'Token created')
```

Comment:
The tiniest of nitpicks, but for the non-JWT token the success message is "New token created", so for consistency's sake:
```suggestion
            messages.success(self.request, 'New token created')
```